bench/workgen/runner/many-dhandle-stress.py

```diff
@@ -90,29 +90,35 @@
 populate_threads = 1
 icount = 15000000
 random_range = 1500000000
+
+# Populate
 pop_ops = Operation(Operation.OP_INSERT, tables[0])
 pop_ops = op_populate_with_range(pop_ops, tables, icount, random_range, populate_threads)
 pop_thread = Thread(pop_ops)
 pop_workload = Workload(context, populate_threads * pop_thread)
 pop_workload.run(conn)
 
+# Insert thread
 ops = Operation(Operation.OP_INSERT, tables[0], Key(Key.KEYGEN_PARETO, 0, ParetoOptions(10)))
 # Updated the range_partition to False, because workgen has some issues with range_partition true.
 # Revert it back after WT-7332.
 ops = op_multi_table(ops, tables, False)
-thread0 = Thread(ops)
-thread0.options.throttle=1000
-thread0.options.throttle_burst=1.0
+insert_thread = Thread(ops)
```

Comment:
This workgen workload was generated from `wtperf/runners/many-dhandle-stress.wtperf`, So I would suggest not changing the thread names. It is understood from the operation (Operation.OP_SEARCH/Operation.OP_INSERT) parameter about the threads.

Reply:
Understood, will revert.

Reply:
Thanks @etienneptl